keras_nlp/tokenizers/tokenizer.py

```diff
@@ -121,5 +122,13 @@ def token_to_id(self, token: str) -> int:
             f"{self.__class__.__name__}."
         )
 
+    def save_to_preset(self, preset):
+        """Save tokenizer to a preset directory.
+
+        Args:
+            preset: The path to the local model preset directory.
+        """
+        save_to_preset(self, preset, config_filename="tokenizer.json")
```

Comment:
Small nit but can do 

```py
from keras_nlp.utils.preset_utils import save_to_preset, TOKENIZER_CONFIG_FILE
```

above and then

```suggestion
save_to_preset(self, preset, config_filename=TOKENIZER_CONFIG_FILE)
```

here